source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,95 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a query document that specifies the
```

Comment:
```suggestion
The ``FindOne()`` method takes a filter document that specifies the
```

This is more consistent with terminology in the [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read) and the [driver documentation](https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo#Collection.FindOne).

Reply:
Thanks for the note. I agree that filter is more consistent with the Go driver documentation. Is it safe to say that going forward, you prefer the terms "filter document" and "filter", over "query document" and "query" when discussing MongoDB queries/filters in the context of the go driver?

Committing suggested change.